### PR TITLE
update requirements

### PR DESCRIPTION
**Pull request by f-peverali**
# Contributor Pull Request

## Description
update requirements for MS reasonCode/reasonReference

## Motivation and Context
ANFISK-323

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this IG / specification.
- [ ] My change requires a change to the documentation or narrative (intend) of the IG.
- [x] I have already updated the documentation / narrative (intend) accordingly.

### CODE REVIEW

**Review by f-peverali — COMMENTED** [review]
suggestion: ggf. noch Hinweis für MedicationRequest (ohne MS)

File: ImplementationGuide/markdown/ReleaseNotes.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ Im Rahmen der ISiK-Veröffentlichungen wird das [Semantic Versioning](https://se
 
 Die erste Ziffer X bezeichnet ein Major-Release und regelt die Gültigkeit von Releases. Die dritte Ziffer Y (Release x.0.y) bezeichnet eine technische Korrektur und versioniert kleinere Änderungen (Packages) während eines Jahres, z. B. 1.0.1.
 
+
+
+Version: 4.0.1
+
+Datum: tbd
+
+* Möglicher Breaking Change für Clients: Änderung (Schwächung) der MS-Definition für .reasonCode und reasonReference für MedicationAdministration und MedicationStatement
```
Comment (f-peverali) [inline ImplementationGuide/markdown/ReleaseNotes.md]:
```suggestion
* Möglicher Breaking Change für Clients: Änderung (Schwächung) der MS-Definition für .reasonCode und reasonReference für MedicationAdministration und MedicationStatement https://github.com/gematik/spec-ISiK-Medikation/pull/140
```

File: Resources/fsh-generated/resources/StructureDefinition-ISiKMedikationsInformation.json [hunk 2]
```diff
@@ -423,6 +424,7 @@
         "id": "MedicationStatement.reasonReference",
         "path": "MedicationStatement.reasonReference",
         "short": "Grund der Medikation (Referenz)",
+        "comment": "  Festlegung zum MS: Die Elemente .reasonCode und .reasonReference MÜSSEN nach OR-Logik in der Ausgabe verwendet werden, d.h. nur eines MUSS geliefert werden können. Weiterhin MÜSSEN beide Elemente interpretiert werden können.\n  \n  Begründung zu Must-Support: Konsolidierung mit MII.",
```
Comment (patrick-werner) [inline Resources/fsh-generated/resources/StructureDefinition-ISiKMedikationsInformation.json]:
```suggestion
        "comment": "  Festlegung zum MS: Die Elemente .reasonCode und .reasonReference MÜSSEN nach OR-Logik in der Ausgabe verwendet werden, d.h. mindestens eines MUSS geliefert werden können. Weiterhin MÜSSEN beide Elemente interpretiert werden können.\n  \n  Begründung zu Must-Support: Konsolidierung mit MII.",
```

**Review by patrick-werner — CHANGES_REQUESTED** [review]
siehe suggestion

**Comment by f-peverali** [comment]
> suggestion: ggf. noch Hinweis für MedicationRequest (ohne MS)

@lhitc könntest du hier bitte deinen Vorschlag für Stufe 4 einbringen?

**Review by lhitc — APPROVED** [review]

File: ImplementationGuide/markdown/ReleaseNotes.md [hunk 3]
```diff
@@ -4,6 +4,15 @@ Im Rahmen der ISiK-Veröffentlichungen wird das [Semantic Versioning](https://se
 
 Die erste Ziffer X bezeichnet ein Major-Release und regelt die Gültigkeit von Releases. Die dritte Ziffer Y (Release x.0.y) bezeichnet eine technische Korrektur und versioniert kleinere Änderungen (Packages) während eines Jahres, z. B. 1.0.1.
 
+
+
+Version: 4.0.1
+
+Datum: tbd
+
+* Möglicher Breaking Change für Clients: Änderung (Schwächung) der MS-Definition für .reasonCode und reasonReference für MedicationAdministration und MedicationStatement https://github.com/gematik/spec-ISiK-Medikation/pull/140
```
Comment (f-peverali) [inline ImplementationGuide/markdown/ReleaseNotes.md]:
```suggestion
* ´improve´ ´test modifying´ Möglicher Breaking Change für Clients: Änderung (Schwächung) der MS-Definition für .reasonCode und reasonReference für MedicationAdministration und MedicationStatement https://github.com/gematik/spec-ISiK-Medikation/pull/140
```

**Pull request merged**